antenna-documentation/scripts/createOrUpdateVersionDocumentationFolder.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+# Copyright (c) Bosch Software Innovations GmbH 2019.
+#
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+BASEDIR=$(readlink -f "$0")
+BASEDIR=$(dirname "$BASEDIR")
+
+set -e
+
+CURRENT_BRANCH=$1
+# checkout branch and build antenna
+git checkout ${CURRENT_BRANCH}
```

Comment:
Why is a `git checkout` necessary at all? If we have this file, we already have a Git working tree. Isn't it good enough to then be able to run this script to create docs form the current branch? Or do we need to be able to create docs for other branches, too, and these other branches also don't have this script yet?

Reply:
no you are right, this is redundant. We just want the documentation of this one branch